pkg/operation/botanist/network/network.go

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Does it make sense to only have one package for extensions into which all related operations are placed? So, something like `/pkg/operation/botanist/extensions/network.go`, `/pkg/operation/botanist/extensions/worker.go`, etc.? Or is there a reason that for a dedicated package per extension kind?

Reply:
No, I ll structure the packages as suggested.